specs/median.test.ts

```diff
@@ -1,15 +1,40 @@
-import { describe, it, expect } from './_describe.ts'
+import { describe, it } from './_describe.ts'
 import { median } from '../mod.ts'
 import { eq } from './utils/utils.ts'
 
 describe('median', () => {
   it('should return middle value of an odd-length list', () => {
     eq(median([2]), 2)
```

Comment:
add some tests with negative values, large values like 10^6, some with decimal point too.
Same for other too